<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Step 1 : Select Processing Mode :</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="singleProcSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>41</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Single Image Processing</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="batchProcSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Batch Processing</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>339</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Step 2 : Threshold Multiplication Factor :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>141</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>ACH Factor :</string>
     </property>
    </widget>
    <widget class="QSlider" name="achSlider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Word Gap Factor :</string>
     </property>
    </widget>
    <widget class="QSlider" name="wordgapSlider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QLabel" name="achValue">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QLabel" name="wordgapValue">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>70</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Step 3 : Execute Table Detection and Retention Algorithm :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="algoFireButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Table Detection and Retention Algorithm</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>761</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Progress :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="progressLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/cdaclogo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/pict logo.gif</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="inputPathLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>26</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="skewCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Skew Correction</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="method2CheckBox">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Method 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="method1CheckBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Method 1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>Input</string>
    </property>
    <addaction name="actionSelect_File"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_Folder"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout_US">
    <property name="title">
     <string>About Us</string>
    </property>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="actionDev_Team"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuAbout_US"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSelect_File">
   <property name="text">
    <string>Select File ...</string>
   </property>
  </action>
  <action name="actionSelect_Folder">
   <property name="text">
    <string>Select Folder ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionDev_Team">
   <property name="text">
    <string>Dev Team</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
